apps/site/pages/docs/api-extensions.mdx

```diff
@@ -54,7 +62,7 @@ starter.store/
 
 Your new type definitions set the data structure for your new fields, extended from the existing FastStore API GraphQL queries and types.
 
-You can create a new `<typeName>.graphql` files inside the `vtex/typeDefs` folder. The way that it will be organized inside this folder is flexible, it's possible to create multiple typeDefs files or group them all together in just one file.
+You can create new `<typeName>.graphql` files inside the `vtex/typeDefs` folder. The way that it will be organized inside this folder is flexible, it's possible to create multiple typeDefs files or group them all together in just one file.
```

Comment:
```suggestion
You can create new `<typeName>.graphql` files inside the `vtex/typeDefs` folder. The way that it will be organized inside this folder is flexible, it's possible to create multiple typeDefs files or group them all together in just one file. During the build process and when running the local server file, all `.graphql` files under this directory will be considered.
```